api/rbody/rbody_test.go

```diff
@@ -1,31 +1,33 @@
-package main
+package rbody
 
 import (
+	"github.com/raintank/metrictank/api/models"
 	"gopkg.in/raintank/schema.v1"
+	"net/http/httptest"
 	"testing"
 )
 
 func TestJsonMarshal(t *testing.T) {
 	cases := []struct {
-		in  []Series
+		in  []models.Series
 		out string
 	}{
 		{
-			in:  []Series{},
+			in:  []models.Series{},
 			out: `[]`,
 		},
 		{
-			in: []Series{
+			in: []models.Series{
 				{
 					Target:     "a",
 					Datapoints: []schema.Point{},
 					Interval:   60,
 				},
 			},
-			out: `[{"Target":"a","Datapoints":[],"Interval":60}]`,
```

Comment:
It was just for graphite-metrictank.  But things have improved and metrictank now handles all of data munging for us like allign requests, merge series etc...